packages/babel-parser/test/fixtures/typescript/class/index-signature/input.ts

```diff
@@ -1,4 +1,6 @@
 class C {
     [x: string]: any;
     readonly [x: string]: any;
+    static [x: string]: any;
+    static readonly [x: string]: any;
```

Comment:
Can you add a test case when `readonly` is before `static`?

```ts
class C {
  readonly static [key: string]: string;
}
```

TS nightly will throw 
> 'static' modifier must precede 'readonly' modifier.